prow/config.yaml

```diff
@@ -22,6 +22,19 @@ tide:
 push_gateway:
   endpoint: pushgateway
 
+# There must be a better way to do this
+all_release_branches: &all_release_branches
+- release-1.0
```

Comment:
Surprise surprise! Now that it is cheap to add technical debt we immediately do so!

Reply:
I believe this is less technical debt that what we have now, but if you feel that copying the exact same few lines dozens of times throughout the config is better, I guess I can revert this.